src/source/Ice/IceAgent.c

```diff
@@ -2299,6 +2326,8 @@ STATUS handleStunPacket(PIceAgent pIceAgent, PBYTE pBuffer, UINT32 bufferLen, PS
 
             CHK_WARN(transactionIdStoreHasId(pIceCandidatePair->pTransactionIdStore, pBuffer + STUN_PACKET_TRANSACTION_ID_OFFSET), retStatus,
                      "Dropping response packet because transaction id does not match");
+            pIceAgent->rtcIceServerDiagnostics[pIceCandidatePair->local->iceServerIndex].totalResponsesReceived++;
```

Comment:
This is a response from the peer. If the connection is p2p there is no ice server involved in this. I think we should only increment count if local candidate is turn

Reply:
Fixed.